### DISCUSSION TITLE
[Note for future] Shift focus to Life Cycle Assessment for the Hardware inputs?

### DISCUSSION THREAD

**Discussion by nabilahmed739** [Green-Software-Foundation/sci#254] [General]
I understand that the current focus is on Carbon, however, in the long term we need to shift our focus from the "carbon tunnel vision" and broaden the vision onto various other impact categories like water use, land use change, abiotic resource depletion and also consider further downstream steps like end-of-life impacts of e-waste, etc.. 
This is more like a note for the future, when we have reached consensus and everyone is working on carbon action and ready for the next step. We need to include life cycle impacts throughout the "digital supply chain", not just for the IT and computing equipment but also for energy production (energy-mix in the region of use, even renewables have impacts) and other auxiliary infrastructure (buildings, HVAC equipment, supporting structures mostly plastics or metal frames - mining too has impacts on the land use change category, biodiversity loss and social impacts). We need to appreciate the use of regenerative and circular materials too. 
Finally, maybe many many years from now when Life Cycle Sustainability Assessment (LCSA*) becomes mainstream the we can have software sustainability assessments covering many more aspects of sustainability rather than just carbon but as a next step including LCA and reporting other relevant impacts could make this more holistic.

* LCSA is defined as jointly performing an environmental life cycle assessment (LCA), an economic life cycle costing (LCC) and a social LCA (SLCA).

**Comment by atg-abhishek** [comment]
Thank you for sharing this @nabilahmed739 - we will bring this up for discussion in this week's WG call. Thanks!

**Comment by Henry-WattTime** [comment]
Notes from WG call:
Daniel: considers effects in other realms, software 
LCA for underlying hardware would be helpful.

Abhishek: carbon was quite difficult alone. Should currently restrict ourselves to carbon. Maybe include other environmental impacts in future versions. (Henry supports).

@nabilahmed739 We really appreciate this suggestion, but currently think that carbon is our primary focus. We may revisit this in future versions.

**Reply by nabilahmed739** [reply]
I understand the priority as well as the challenge. Hopefully we can come to including LCA of the hardware atleast. 
Good luck team!
